src/_pytest/terminal.py

```diff
@@ -1055,3 +1056,12 @@ def _plugin_nameversions(plugininfo):
         if name not in values:
             values.append(name)
     return values
+
+
+def format_session_duration(seconds):
+    """Format the given seconds in a human readable manner to show in the final summary"""
+    if seconds < 60:
+        return "{:.2f}s".format(seconds)
+    else:
+        dt = datetime.timedelta(seconds=int(seconds))
+        return "{:.2f}s ({})".format(seconds, dt)
```

Comment:
Does not really need the "else".

Reply:
I'm aware, I usually like to be explicit thought (it is a matter of style after all).